### ISSUE TITLE
BT-120 (No Negotiation Necessary) Isn't this term rather about possibility instead of necessity?

### ISSUE THREAD

**Issue by ColinMaudry** [eForms/eForms#323]
The term BT-120 is labelled "No Negotiation Necessary" and its description is

> The buyer reserves the right to award the contract on the basis of the initial tenders without any further negotiations.

The **description** expresses the fact that the buyer reserves the right, the possibility, to cut off the negotiations, while I feel the **label** expresses the fact that the buyer considers, at Contract Notice stage, that the negotiations are not necessary and will consequently not take place.

I believe the semantics documented in the description are the ones intended to be expressed with this term. Shouldn't the label be replaced with something like "Possibly No Negotiation"?

**Comment by jpmckinney** [comment]
I think "not necessary" is to be interpreted as "not required" or "not mandatory", i.e. "optional".

However, "optional" expresses more of a 'positive' right like "I have the option to negotiate if I want", whereas this is more of a 'negative' right i.e. "I have the option to *not* negotiate if I want". 

That's my understanding of why "No Negotiation Necessary" was chosen.

**Comment by ColinMaudry** [comment]
FYI, we chose to label this term `mayNotHaveNegotiation` in the Second stage OCDS extension.

**Comment by JachymHercher** [comment]
1) I'm wondering whether the nuance is worth having different terms? It doesn't matter much, but still. 

2) Also, isn't "mayNotHaveNegotiation" ambiguous? In the sense of mayNot either being "it is possible that he doesn't have" or "he is forbidden to have"? 

3) Finally, just FYI: in eForms, the field exists only in the forms under the classical directive, not because under the other ones you couldn't "skip negotiation", but because in the other ones you can always skip negotiation and you don't need to even reserve the right.

**Comment by ColinMaudry** [comment]
2. Is there a case where negotiation could be forbidden?
3. What is the "classical" directive you refer to? Is there a romantic directive? :smile: Seriously, I'd be grateful if you could develop this point.

**Comment by JachymHercher** [comment]
2. The open procedure is without negotiation. If you're in a negotiated procedure then no, it can't be forbidden. Which doesn't mean it couldn't seem confusing though :-). 
3. :-), sorry for the jargon. The classical Directive is Directive 2014/24/EU, as compared to the "utilities" Directive 2014/25/EU, the "concessions" Directive 2014/23/EU and the "Defence" directive 2009/81/EC.

**Comment by ColinMaudry** [comment]
Thanks!

**Comment by jpmckinney** [comment]
Since this is a narrow property which might only be needed in the EU, I'm fine with the original name. Otherwise, mightNotHaveNegotiation would address (2), which we should address. I can't think of a brief name that more directly contains the desired semantics.

**Comment by ColinMaudry** [comment]
To be sure my point of view is clear, for the fun of discussing semantics and fully considering the fact that English is not my mother tongue, here are my arguments against the original term: `noNegotiationNecessary`.

I looked for a common and similar phrasing, and found "No purchase necessary". In the context of [sweepstakes](https://www.thebalanceeveryday.com/no-purchase-necessary-to-enter-definition-4126050), the giveaways organized by a company that look like a lottery, it means that the participants are not bound to purchase a product or service to the company to participate to the giveaway.

That gives us:

- In a given giveaway, the participant doesn't need to purchase something to participate
- In a `noNegotiationNecessary` contracting process, the buyer doesn't need to enter negotiations to award the contract

This little comparison shows that `noNegotiationNecessary` is indeed used to convey the intended meaning. I still believe that it may convey be understood (as I did) as "the buyer doesn't need negotiations the same way a healthy person doesn't need a wheelchair, and will consequently not bother negotiating".

Given `noNegotiationNecessary` works and we couldn't really agree on a better name, I think OCDS and eForms should use the same term for the good of those who will compare them.

Concisely and unambiguously naming stuff is hard!
